Resources/Prototypes/Reagents/Consumable/Food/ingredients.yml

```
@@ -120,9 +120,15 @@
       - !type:AdjustReagent
         reagent: Water
         amount: 0.4
-      - !type:AdjustReagent
-        reagent: Vitamin
-        amount: 0.1
+#      - !type:AdjustReagent # Frontier
+#        reagent: Vitamin # Frontier
+#        amount: 0.1 # Frontier
+      - !type:SatiateHunger # Frontier
+        conditions: # Frontier
+        - !type:ReagentThreshold # Frontier
+          reagent: Nutriment # Frontier
+          min: 0.1 # Frontier
+        factor: 1
```

Use a factor of 0.2 here - vinegar isn't particularly filling, and it was 20% vitamin to start with.
```suggestion
        factor: 0.2
```